assets/css/custom.css

```
@@ -1,440 +1,8 @@
-   A cohesive, modern, ergonomic dark "midnight" aesthetic.
-   ============================================================ */
 
-:root {
-  /* Type system — Wix Madefor (brand decision 2026-08-21) */
-  --font-sans: "Wix Madefor Text", "WixMadeforText", ui-sans-serif, system-ui,
-    -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans",
-    sans-serif;
-  --font-mono: "Wix Madefor Mono", "JetBrains Mono", ui-monospace,
-    "SFMono-Regular", "Menlo", "Consolas", "Liberation Mono", monospace;
-  --content-width: 46rem;
+z-index:1;overflow:hidden;opacity:0;transition:opacity .35s ease}
```

Stylelint (17.14.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)